introduction.md

```diff
@@ -1 +1,5 @@
 Hi, this is TCBarrett. My favourite code school path is HTML & CSS
+
+Deadly skills:
+ * GIT
+ * Standing jumps
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?